uli_init/simulate.py

```diff
@@ -215,8 +229,27 @@ def quench(
                 dynamic=["momentum"]
             )
             # Run the primary simulation
-            integrator.set_params(kT=kT)
-            integrator.randomize_velocities(seed=self.seed)
+            if pressure:
+                try: # Not defined if no shrink step
+                    integrator.disable() 
+                except NameError:
+                    pass
+                integrator = hoomd.md.integrate.npt(
+                        group=_all,
+                        tau=self.tau_kt,
+                        tauP=self.tau_p,
+                        P=pressure,
+                        kT=kT
+                        )
+            elif not pressure:
+                try:
+                    integrator.__getattribute__
```

Comment:
what is this line doing? This is to handle if no shrink steps are run, correct?
```suggestion
                    integrator
```
If you are just trying to check if it exists, this works as well

Reply:
duh :upside_down_face: